README.md

```diff
@@ -6,22 +6,333 @@ Programming (MIP) problems.
 ## License
 
 Please note that go-mip is provided as _source-available_ software (not
-_open-source_). For further information, please refer to the [LICENSE](./LICENSE.md)
-file.
+_open-source_). For further information, please refer to the
+[LICENSE](./LICENSE.md) file.
 
 ## Usage
 
 This library contains the types and interfaces to model and solve MIP problems.
 The implementation is provided by other (solver-specific) packages, e.g.:
 [go-highs](https://github.com/nextmv-io/go-highs).
 
-For further information on how to get started with MIP modeling and Nextmv,
-please refer to the [official documentation](https://docs.nextmv.io/docs/mixed-integer-programming).
+These are the supported solvers and solver interfaces for MIP:
 
-## Versioning
+| Provider | Description |
+| -------- | ----------- |
+| [FICO Xpress][fico-xpress-section] | Commercial solver. |
+| [HiGHS][highs-section] | Open-source solver.  |
+| [OR-Tools][ortools-section] | Open-source solver interface for various solvers. Included open-source solvers: `CLP`, `GLOP`, `PDLP`, `SCIP`. |
+| [Pyomo][pyomo-section] | Open-source interface for various solvers. Included open-source solvers in Cloud: `CBC`, `GLPK`. |
```

Comment:
This repo doesn’t know about OR-Tools and Pyomo, so we need to eliminate all mentions of them.